UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/Data/Gas.cs

```diff
@@ -12,7 +12,7 @@ public struct Gas
 		public static readonly Gas Plasma = new Gas("Plasma", 200, 40f, true, 0.4f, "PlasmaAir", -4, 0);
 		public static readonly Gas Oxygen = new Gas("Oxygen", 20, 31.9988f, false, 0.4f, "NONE", -3, 0);
 		public static readonly Gas Nitrogen = new Gas("Nitrogen", 20, 28.0134f, false, 0.4f, "NONE", -3, 0);
-		public static readonly Gas CarbonDioxide = new Gas("Carbon Dioxide", 30, 44.01f, false, 0.4f, "NONE", -3, 0);
+		public static readonly Gas CarbonDioxide = new Gas("Carbon Dioxide", 20, 44.01f, false, 0.4f, "NONE", -3, 0);
```

Comment:
I'm guessing you made this change while testing issue #5132? Could probably remove this change?

https://i.imgur.com/xLMGVzS.png